lib/smart_answer_flows/simplified-expenses-checker.rb

```diff
@@ -119,10 +119,10 @@ def define
         next_node do |response|
           calculator.vehicle_costs = response
 
-          if calculator.car? || calculator.van?
-            question :drive_business_miles_car_van?
-          else
-            question :drive_business_miles_motorcycle?
+          if calculator.new_car? || calculator.used_car?
```

Comment:
Just keeping in line with the defined logic.. `if calculator.car?` This is just a lot more explicit given the context from the logic document